lib/Adapter/Composer/ComposerClassToFile.php

```diff
@@ -54,30 +48,40 @@ private function getStrategies(): array
             [
                 $this->classLoader->getPrefixesPsr4(),
                 new Psr4NameInflector(),
+                Psr4NameInflector::NAMESPACE_SEPARATOR,
             ],
             [
                 $this->classLoader->getPrefixes(),
                 new Psr0NameInflector(),
+                Psr0NameInflector::NAMESPACE_SEPARATOR,
             ],
             [
                 $this->classLoader->getClassMap(),
                 new ClassmapNameInflector(),
+                Psr4NameInflector::NAMESPACE_SEPARATOR,
             ],
             [
                 $this->classLoader->getFallbackDirs(),
                 new Psr0NameInflector(),
+                Psr0NameInflector::NAMESPACE_SEPARATOR,
             ],
             [
                 $this->classLoader->getFallbackDirsPsr4(),
                 // PSR0 name inflector works here as there is no prefix
                 new Psr0NameInflector(),
+                Psr0NameInflector::NAMESPACE_SEPARATOR,
```

Comment:
hm, why do we pass this in if it's the same each time?

Reply:
Not the same: `Psr0NameInflector::NAMESPACE_SEPARATOR` and `Psr4NameInflector::NAMESPACE_SEPARATOR` are used as an additional separator. 

Such separator would be returned from `NameInflector::getAdditionalSeparator()` or something similar and the we would not need pass it there. If you agree then I can make this change.

Reply:
ah ok, confused though why would `\\` be an additional separator for PSR4? but yeah if we could improve the intent of these constnats/and methods it would be good :+1: 

now that we've merged a PR the CI should run for you by default... :crossed_fingers:

Reply:
https://github.com/phpactor/class-to-file/pull/27

Reply:
Please do not forget to make a release and upgrade it in phpactor while it's not a part of the monolith.

Reply:
Thanks!